parlai/core/opt.py

```diff
@@ -25,7 +25,7 @@
     "batchindex",
     "download_path",
     "datapath",
-    "batchindex",
```

Comment:
Lol don't kill this!

Reply:
it's not. there is another  "batchindex" on L25